exonum-java-binding/qa-service/src/main/java/com/exonum/binding/qaservice/TimeDto.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2019 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exonum.binding.qaservice;
+
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Objects;
+import java.time.ZonedDateTime;
+
+/**
+ * DTO class for JSON view.
+ */
+final class TimeDto {
+  private final ZonedDateTime time;
+
+  TimeDto(ZonedDateTime time) {
+    this.time = time;
+  }
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
Is it ok to leave these methods out for dtos? Or let them remain here?

Reply:
I introduced those as I compared DTOs in tests and needed these methods, but now we don't do that 🤷‍♂️ Still I'd leave them be, why not - in case we ever need to compare DTOs. But can remove :)